src/sia/models/sia_query_params.py

```diff
@@ -0,0 +1,411 @@
+    target : Optional[list[str]]
+        Name of the target.
+    collection : Optional[list[str]]
+        Name of the data collection.
+    facility : Optional[list[str]]
+        Name of the facility.
+    instrument : Optional[list[str]]
+        Name of the instrument.
+    maxrec : Optional[int]
+        Maximum number of records in the response.
+    responseformat : Optional[str]
+        Format of the response.
+
+    Notes
+    -----
+    I have tried using Pydantic here, but as I understand there is currently
+    a limitation with how FastAPI handles Pydantic models for query parameters
+    with list attributes
+    (See: https://github.com/fastapi/fastapi/discussions/10556)
+    """
+
+    pos: Annotated[
+        list[str] | None,
+        Query(
+            title="pos",
+            description="Positional region(s) to be searched",
+            examples=["320 -0.1 10"],
+        ),
+    ] = None
+
+    q_format: Annotated[
+        list[str] | None,
+        Query(
+            title="format",
+            alias="format",
+            description="Response format(s)",
+            examples=["application/x-votable+xml"],
+        ),
+    ] = None
+
+    time: Annotated[
+        list[str] | None,
+        Query(
+            title="time",
+            description="Time interval(s) to be searched",
+            examples=["2021-01-01T00:00:00Z 2021-01-02T00:00:00Z"],
+        ),
+    ] = None
+
+    band: Annotated[
+        list[str] | None,
+        Query(
+            title="band",
+            description="Energy interval(s) to be searched",
+            examples=["0.1 10.0"],
+        ),
+    ] = None
+
+    pol: Annotated[
+        list[Polarization] | None,
+        Query(
+            title="pol",
+            description="Polarization state(s) to be searched",
+            examples=["I", "Q"],
+        ),
+    ] = None
+
+    fov: Annotated[
+        list[str] | None,
+        Query(
+            title="fov",
+            description="Range(s) of field of view",
+            examples=["1.0 2.0"],
+        ),
+    ] = None
+
+    spatres: Annotated[
+        list[str] | None,
+        Query(
+            title="spatres",
+            description="Range(s) of spatial resolution",
+            examples=["0.1 0.2"],
+        ),
+    ] = None
+
+    exptime: Annotated[
+        list[str] | None,
+        Query(
+            title="exptime",
+            description="Range(s) of exposure times",
+            examples=["-Inf 60"],
+        ),
+    ] = None
+
+    timeres: Annotated[
+        list[str] | None,
+        Query(
+            title="timeres",
+            description="Range(s) of temporal resolution",
+            examples=["-Inf 1.0"],
+        ),
+    ] = None
+
+    specrp: Annotated[
+        list[str] | None,
+        Query(
+            title="specrp",
+            description="Range(s) of spectral resolving power",
+            examples=["1000 2000"],
+        ),
+    ] = None
+
+    q_id: Annotated[
```

Comment:
You can just use `id`, Python figures it out. There are only a few keywords that can't be used for identifiers, and we whitelist that one for Ruff checks because it's so common to want to use it.